tests/boxed.rs

```diff
@@ -64,4 +64,17 @@ compile_pass! {
             global.gc();
         }
     ),
+    (
+    export class Api {
+        constructor();
+
+        static fetch(url, imports): Promise<void>;
+
+        make_box(value: number): CustomType;
+
+        do_something(boxed: CustomType): number;
```

Comment:
here the argument would be box

Reply:
How could I deduce that?

Reply:
never mind, removed support for raw boxes